liota/entities/metrics/registered_metric.py

```diff
@@ -30,39 +30,102 @@
 #  THE POSSIBILITY OF SUCH DAMAGE.                                            #
 # ----------------------------------------------------------------------------#
 
-from liota.entities.metrics.metric import Metric
 from Queue import Queue
+import inspect
+import logging
+
+from liota.core import metric_handler
+from liota.entities.metrics.metric import Metric
+from liota.entities.registered_entity import RegisteredEntity
+from liota.lib.utilities.utility import getUTCmillis
 
 
-class RegisteredMetric():
+log = logging.getLogger(__name__)
+
+
+class RegisteredMetric(RegisteredEntity):
 
     def __init__(self, ref_metric, ref_dcc, reg_entity_id):
         if not isinstance(ref_metric, Metric):
             raise TypeError
-        super(self.__class__, self).__init__(
-            ref_entity=ref_metric,
-            ref_dcc=ref_dcc,
-            reg_entity_id=reg_entity_id
-        )
+        RegisteredEntity.__init__(self,
```

Comment:
What is the advantage of using `ParentClass.__init__(self)` against `super(self.__class__, self).__init__()`?